aws_lambda_powertools/utilities/trigger/api_gateway_proxy_event.py

```diff
@@ -2,205 +2,205 @@
+        return self._v["requestContext"]["identity"].get("cognitoAuthenticationType")
 
     @property
     def cognito_identity_id(self) -> Optional[str]:
         """The Amazon Cognito identity ID of the caller making the request.
         Available only if the request was signed with Amazon Cognito credentials."""
-        return self._val["requestContext"]["identity"].get("cognitoIdentityId")
+        return self._v["requestContext"]["identity"].get("cognitoIdentityId")
 
     @property
     def cognito_identity_pool_id(self) -> Optional[str]:
         """The Amazon Cognito identity pool ID of the caller making the request.
         Available only if the request was signed with Amazon Cognito credentials."""
-        return self._val["requestContext"]["identity"].get("cognitoIdentityPoolId")
+        return self._v["requestContext"]["identity"].get("cognitoIdentityPoolId")
 
     @property
     def principal_org_id(self) -> Optional[str]:
         """The AWS organization ID."""
-        return self._val["requestContext"]["identity"].get("principalOrgId")
+        return self._v["requestContext"]["identity"].get("principalOrgId")
 
     @property
     def source_ip(self) -> str:
         """The source IP address of the TCP connection making the request to API Gateway."""
-        return self._val["requestContext"]["identity"]["sourceIp"]
+        return self._v["requestContext"]["identity"]["sourceIp"]
 
     @property
     def user(self) -> Optional[str]:
         """The principal identifier of the user making the request."""
-        return self._val["requestContext"]["identity"].get("user")
+        return self._v["requestContext"]["identity"].get("user")
 
     @property
     def user_agent(self) -> Optional[str]:
         """The User Agent of the API caller."""
-        return self._val["requestContext"]["identity"].get("userAgent")
+        return self._v["requestContext"]["identity"].get("userAgent")
 
     @property
     def user_arn(self) -> Optional[str]:
         """The Amazon Resource Name (ARN) of the effective user identified after authentication."""
-        return self._val["requestContext"]["identity"].get("userArn")
+        return self._v["requestContext"]["identity"].get("userArn")
 
 
 class APIGatewayEventAuthorizer:
-    def __init__(self, event: Dict):
-        self._val = event
+    def __init__(self, event: Dict[str, Any]):
+        self._v = event
 
     @property
     def claims(self) -> Optional[Dict[str, Any]]:
-        return self._val["requestContext"]["authorizer"].get("claims")
+        return self._v["requestContext"]["authorizer"].get("claims")
 
     @property
     def scopes(self) -> Optional[List[str]]:
-        return self._val["requestContext"]["authorizer"].get("scopes")
+        return self._v["requestContext"]["authorizer"].get("scopes")
 
 
 class APIGatewayEventRequestContext:
     def __init__(self, event: Dict[str, Any]):
-        self._val = event
+        self._v = event
```

Comment:
Why not make this protected attribute as `_event` instead of `_v`? I recommend refactor this attribute to `_event`

Reply:
I just wanted short and consistent. Maybe I can have a Abstract class for all data classes that has __getitem__ implemented

Reply:
@koiker  - i created a `DictWrapper` when replaces this.